docs/usage/verticles/ServerVerticle.md

```diff
@@ -0,0 +1,183 @@
+# ServerVerticle
+
+The `ServerVerticle` is a Vert.x verticle that exposes endpoints using the HTTP(S) protocol. NeonBee deploys the
+`ServerVerticle` automatically if the `WEB` [profile](./../neonbee.md#neonbee-profiles) is active. When the
+`ServerVerticle` is started, it creates a router and mounts the specified endpoints onto it. It then creates an HTTP
+server using the router. When the `ServerVerticle` is stopped, the HTTP server is closed and additional logging is
+performed.
+
+To use the `ServerVerticle`, you will need to provide it with a configuration object in the form of a JSON object. The
```

Comment:
You will never deoploy your own ServerVerticle, there MUST only one per instance. So I would explain the yaml/json config only and don't talk about a JsonObject which is passed by NeonBee.